config/environments/production.rb

```diff
@@ -20,7 +20,7 @@
   # config.action_dispatch.rack_cache = true
 
   # Disable Rails's static asset server (Apache or nginx will already do this).
-  config.serve_static_assets = false
+  config.serve_static_assets = ENV["RAILS_SERVE_STATIC_ASSETS"] =~ /(true|yes)/i
```

Comment:
It feels like this line and the ones below could benefit from a couple of comments explaining purpose. Without the Heroku context they might be a bit opaque.